### DISCUSSION TITLE
eksctl supported eks cluster?

### DISCUSSION THREAD

**Discussion by andrei-on-github** [weaveworks/eksctl#6240] [Q&A]
Hello.

I'm using eksctl to create an EKS cluster today 05.02.2023. As soon as eksctl created the cluster I went into the AWS Console where I saw a message that my cluster version is not supported, and to update my cluster version.
The command I used to create the cluster and my eksctl version: 
$ eksctl create cluster --name training --version 1.20 --with-oidc --ssh-public-key <redacted>
eksctl version:0.122.0
The official documentation[1] for AWS EKS support shows that eks 1.20 is EOS :November 1, 2022

So my question is why does eksctl allow the creation of new cluster with unsupported versions?
If it's by design I would appreciate an answer to understand the reasoning?

Thank you in advance!

[1] https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar

**Comment by cPu1** [comment]
@andrei-i-florentin, a more recent version of eksctl will prevent you from creating 1.20 clusters. Version-related changes require an update to eksctl.
